models/database.py

```diff
@@ -31,13 +32,15 @@ class User(Base, UserMixin):
     lastname = Column(String(40), nullable=False)
     email = Column(String(120), nullable=False)
     image_file = Column(String(240), nullable=False, default="default.jpg")
-    password = Column(String(60), nullable=False)
+    password = Column(Text, nullable=False)
+    #location = Column(Geography(geometry_type='POINT', srid=4326))
+    #domicile_geolocation = Column(Geography(geometry_type='POINT', srid=4326))
     messages = relationship("Messages", backref="author", lazy=True)

```

Comment:
I'm not sure how you want to handle messages but I recommend adding ChatRoom/Member models